js/app.js

```diff
@@ -0,0 +1,75 @@
+/*
+let answer = {
+    moreAnswers: [1,2,3,4,5],
+    theresMore: {
+        but: "wait",
+    },
+    manKind: function () {
+        console.log("Nay, I am the world")
+    }
+}
+
+//console.log(answer.moreAnswers[2])
+
+//console.log(answer.theresMore.but);
+
+//answer.manKind();
+
```

Comment:
ha i love the entertaining names of things.  
it looks like everything above here was the three you picked for the first question. it's a little unclear which questions you were trying to answer....